components/tools/OmeroPy/src/omero/plugins/prefs.py

```diff
@@ -93,6 +94,45 @@ def with_rw_config(func):
     return wraps(func)(_make_open_and_close_config(func, False))
 
 
+class _OmeroPropertiesFileObj(object):
```

Comment:
Can you unify this with the existing code?

 * https://github.com/openmicroscopy/openmicroscopy/blob/08e8a91b480bccb6e5bc181baa172fdd16f9924a/components/tools/OmeroPy/src/omero/plugins/prefs.py#L463
 * https://github.com/openmicroscopy/openmicroscopy/blob/08e8a91b480bccb6e5bc181baa172fdd16f9924a/components/tools/OmeroPy/src/omero/install/config_parser.py#L231